src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -56,17 +59,18 @@ private static void setupCommandTests(Application application) {
 	 * @param application Application context.
 	 */
 	private static void setupDrivers(Application application) {
-		Job2dDriver loggerDriver = new LoggerDriver();
+		Job2dDriver loggerDriver = new UsageDecorator( new LoggerDriver(), usageLogger );
+
 		DriverFeature.addDriver("Logger driver", loggerDriver);
 
 		DrawPanelController drawerController = DrawerFeature.getDrawerController();
-		Job2dDriver driver = new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic");
+		Job2dDriver driver = new UsageDecorator( new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic"), usageLogger );
```

Comment:
Changing this code makes the future merging more difficult.
UsageDecorator could be applied optionally by addtional menu option:
- statically (using getCurrentDriver and setCurrentDriver)
- dynamically (observing setCurrentDriver notifications)